packages/babel-plugin-transform-typescript/README.md

```diff
@@ -4,8 +4,17 @@
 
 Does not type-check its input. For that, you will need to install and set up TypeScript.
 
-Does not support `namespace`s or `const enum`s because those require type information to transpile.
-Also does not support `export =` and `import =`, because those cannot be transpiled to ES.next.
+## Caveats
+
+* Does not support [`namespace`][namespace]s.
+* Does not support [`const enum`][const_enum]s because those require type information to transpile.
+* Does not support [`export =`][exin] and [`import =`][exin], because those cannot be transpiled to ES.next.
+
+## Workarounds
+
+* `namespace`: Migrate to defining modules instead.
+* `const enum`: Avoid to using enums outside of current file
```

Comment:
You can just change to `enum` (without `const`), which is available as a value at runtime and can be transpiled without type information.

Reply:
Thanks @andy-ms - updated 👍